manifesto.cs.md

```diff
@@ -0,0 +1,11 @@
+Pouze šťastní vývojáři vytvářejí vyjímečné produkty.
+Vlastním vývojem a pomáháním ostatním při implementaci aplikací jsme dospěli k těmto hodnotám:
+
+- **Automatizace opakujících se úkolů** před manuální prací
+- **Standardní používání nástrojů** před nadměrnými konfiguracemi
+- **Dostatečný klid na vývoj** před konstantním tlakem na termíny
+- **Přijímání nejlepších postupů** před dlouho zaběhlými praktikami
+
+Všechny části zmíněných přístupů dávají smysl, ale ty zvýrazněné přináší hodnotu.
```

Comment:
Na monitorech by mělo být
```suggestion
Jakkoliv jsou body napravo hodnotné, bodů nalevo si ceníme více.
```

Na mobilech by mělo být:
```suggestion
Všechny body jsou hodnotné, avšak zvýrazněných bodů si ceníme více.
```

Inspirováno:
https://agilemanifesto.org/iso/cs/manifesto.html

Reply:
Takhle by se to mělo chovat i v anglické verzi.